apps/web/prisma/schema.prisma

```diff
@@ -565,6 +572,19 @@ model DraftSendLog {
   @@index([executedActionId])
 }
 
+model Digest {
```

Comment:
So this I think we would need to mark off if a digest item has been sent or not.

The sort of flow I imagined was email comes in. If there's a digest action we create a digest record for it. Then when the digest email is sent we mark it as handled.

Reply:
Possible we don't even need this, because we have the ExecutedAction with Digest type anyway